### DISCUSSION TITLE
2022-9-20

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#9] [Show and tell]
Win32 API강의 정주행 완료
---
스크린샷을 첨부하고 싶었지만 하는방법을 모르겠다..

총 76강중 71강까지 들었고 해결하지 못한 이슈가 있다.

해결못한 이슈는 Git Hub의 Issues에 남기고 싶었으나 내컴퓨터에 저장된 스샷은 어떻게 올리는지 모르겠음.

Link: [Win32 API 무료강의][link]

[link]: https://youtube.com/playlist?list=PL4SIC1d_ab-ZLg4TvAO5R4nqlJTyJXsPK

Win32 브랜치를 만들어 보았다.
---
jonjin의 하위 브랜치로 생성하였다. 뭔가 더 만들지는 생각 중이라 작업 내용을 진행시키고 커밋 해보는건 고민해봐야할듯

2D Sprite 제작
---
토깽이의 점프 모션이 만들어졌다. 매우 힘든 작업이였는데 여기 스샷을 못올리네

![음?](https://pbs.twimg.com/media/FBo3dzIUYAM-m-Q?format=png&name=small)

**Comment by Woon-2** [comment]
이미지는 `Drag & Drop`으로 쉽게 올릴 수 있습니다.

![이미지](https://user-images.githubusercontent.com/73771162/191405153-97b88a7d-fc7f-44ba-a364-bfed6961df4e.PNG)

`Drag & Drop`을 하면 본문에는 위와 같이 이미지를 표시하기 위한 링크가 생성되고,   
`Preview` 버튼을 눌러보면 해당 이미지가 어떻게 보이는지 알 수 있습니다.

`소괄호()` 안에 있는 부분이 이미지 `URL`인데, 이것을 이용해서 또 다양한 시도를 할 수 있습니다.

![가운데](https://user-images.githubusercontent.com/73771162/191405902-0ba7f1ee-4844-4827-8f83-79a2998aa3eb.PNG)

- `div` 태그와 `image` 태그를 이용해 이미지를 가운데 정렬하는 방법입니다.

![크기](https://user-images.githubusercontent.com/73771162/191405968-9be02ede-bfef-4d00-b0d2-d86f5dd13b4a.PNG)
![가로](https://user-images.githubusercontent.com/73771162/191405970-1e92ae65-aee0-4f17-a99a-72c23433809b.PNG)
![세로](https://user-images.githubusercontent.com/73771162/191405971-07362db7-3f1f-4f00-b0ac-98b30d865c1c.PNG)

- `image` 태그와 `width`, `height` 속성을 이용해 이미지의 크기를 조절하는 방법입니다.

![가운데와 크기](https://user-images.githubusercontent.com/73771162/191406040-c666ee7c-44e7-431b-9970-4c159c61834a.PNG)

- 가운데 정렬과 크기 조절을 동시에 할 경우 위와 같은 방법으로 합니다.
